src/containers/MyAccount/MyAccount.js

```diff
@@ -1,25 +1,19 @@
-import React, { Component } from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 
-class MyAccount extends Component {
-  render() {
-    const { emails } = this.props;
-
-    return (
-      <div style={{ padding: '20px' }} id="my-account-page">
-        <h2>My Account</h2>
-
-        <br/><br/>
-        Email Address: { (emails && emails.length > 0) ? emails[0] : "No email address available."}
-        <br/>
-      </div>
-    );
-  }
-}
+const MyAccount = ({ emails }) => (
```

Comment:
Personally I prefer container components as classes (since they often have some life cycle methods), but this is ok for now.